utils/create-update-packages.sh

```diff
@@ -196,6 +196,8 @@ function do_felix {
     # RPM golang build dependencies that is exactly equivalent to our
     # containerized builds.
     make bin/calico-felix
+    rm -f bin/calico-felix-amd64
```

Comment:
What about `rm -f !(bin/calico-felix)` instead? That better fits the purpose of what we want there, right?
I'd also appreciate a comment like `Clean everything from the build except the binary`.

Reply:
Is that (with the `!`) a real `rm` syntax?  I've never seen it before, and couldn't find it just now in the rm manpage.

Happy to add that comment though...

Reply:
That would just be shell. Oh oops, I had looked that up and I missed that you would need to also do `shopt -s extglob` to enable extglob.
So yeah not `rm` syntax.

Reply:
Thanks, I've found that now too in the bash manpage - nice!

It wouldn't do what I wanted, though, because it would also remove all of the Git source code (e.g. *.go), and I only want to remove build products (except bin/calico-felix).

Reply:
Oh you're right. What I was intending was to remove all from `bin/` except calico-felix.